radarr/naming.go

```diff
@@ -0,0 +1,66 @@
+package radarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"golift.io/starr"
+)
+
+// Define Base Path for Naming calls.
+const bpNaming = APIver + "/config/naming"
+
+// Naming represents the config/naming endpoint in Radarr.
+type Naming struct {
+	RenameMovies             bool   `json:"renameMovies,omitempty"`
+	ReplaceIllegalCharacters bool   `json:"replaceIllegalCharacters,omitempty"`
+	IncludeQuality           bool   `json:"includeQuality,omitempty"`
+	ReplaceSpaces            bool   `json:"replaceSpaces,omitempty"`
+	ID                       int64  `json:"id"` // ID must always be 1 (Oct 10, 2022)
+	ColonReplacementFormat   string `json:"colonReplacementFormat,omitempty"`
```

Comment:
do you want to start using enum here as well? honestly, I think we can avoid it.
for your reference, values are: `delete`, `dash`, `spaceDash`, and `spaceDashSpace`

Reply:
I like the idea of enums, but I'm not married to the idea of putting on everything.